### PR TITLE
Бутеску - Лабораторная работа #1

### PR DESCRIPTION
**Pull request by Dragnsnom**

### CODE REVIEW

**Comment by Dragnsnom** [comment]
@Kirill1210 
@PavelKitaev

**Comment by PavelKitaev** [comment]
Все хорошо

**Comment by Kirill1210** [comment]
Все хорошо

File: modules/complex-number/test/test_Butescu_Vladimir_complex_number.cpp [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+    // Arrange
+    double re = 1.0;
+    double im = 2.0;
+
+    // Act
+    ComplexNumber z(re, im);
+
+    // Assert
+    EXPECT_EQ(1.0, z.getRe());
+    EXPECT_EQ(2.0, z.getIm());
```
Comment (SHVlader) [inline modules/complex-number/test/test_Butescu_Vladimir_complex_number.cpp:16]: The data type "double" wants to be checked in EXPECT_DOUBLE_EQ
Please correct all test

**Comment by SHVlader** [comment]
Don't forget about approved
Pull Request -> File chenged -> Review changes -> Approve -> Submit review

**Review by PavelKitaev — APPROVED** [review]

**Review by Dragnsnom — COMMENTED** [review]
ADD TESTS

**Review by Dragnsnom — COMMENTED** [review]
fix

**Review by bloodreina5 — APPROVED** [review]

**Review by maslichko — APPROVED** [review]

**Pull request merged**